### PR TITLE
Create weekly-update.2015-05-08.md

### PR DESCRIPTION
**Pull request by Gioyik**
CC #68

### CODE REVIEW

File: weekly-updates/weekly-update.2015-05-08.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# io.js 2.0 releases
+This week we had two io.js releases [v2.0.0](https://iojs.org/dist/v2.0.0/) and [v2.0.1](https://iojs.org/dist/v2.0.1/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md).
+
+### Notable changes
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-05-08.md:4]: IMO, we should not merge changelog v2.0.0 and v2.0.1. We should write the changes on each version. 
like: https://github.com/iojs/evangelism/blob/master/weekly-updates/weekly-update.2015-03-20.md

Comment (Gioyik) [inline weekly-updates/weekly-update.2015-05-08.md:4]: Done!

**Comment by Gioyik** [comment]
Changes made!

**Comment by yosuke-furukawa** [comment]
LGTM

**Comment by yosuke-furukawa** [comment]
Merged!

**Comment by julianduque** [comment]
Published: https://medium.com/node-js-javascript/io-js-week-of-may-8th-f9ddd8d3080c

**Pull request merged**